samples/excel/Basics/range.yaml

```diff
@@ -27,27 +53,76 @@ script:
             let sheet = context.workbook.worksheets.getActiveWorksheet();
 
             let data = [
-                ["Product", "Price", "Tax"],
-                ["Apple", 1.99, "=R[0]C[-1] * .09"],
-                ["Oranges", 2.49, "=R[0]C[-1] * .09"]
+                ["Total Price"],
+                ["=C3 * D3"],
+                ["=C4 * D4"],
+                ["=C5 * D5"],
+                ["=SUM(E3:E5)"]
+            ]
+            let range = sheet.getRange("E2:E6");
+            range.formulas = data;
+            range.format.autofitColumns();    
+
+            await context.sync();
+
+        }).catch(OfficeHelpers.Utilities.log);
+    }
+
+    function setFormulasR1C1() {
+        Excel.run(async (context) => {
+            let sheet = context.workbook.worksheets.getActiveWorksheet();
+
+            let data = [
+                ["Total Price"],
+                ["=R[0]C[-2] * R[0]C[-1]"],
+                ["=R[0]C[-2] * R[0]C[-1]"],
+                ["=R[0]C[-2] * R[0]C[-1]"],
+                ["=SUM(R[-3]C[0]:R[-1]C[0])"]
             ]
-            sheet.getCell(0, 0)
-                .getResizedRange(data.length - 1, data[0].length - 1)
-                .formulasR1C1 = data;
+            let range = sheet.getRange("E2:E6");
+            range.formulasR1C1 = data;
+            range.format.autofitColumns();    
+
+            await context.sync();
+
+        }).catch(OfficeHelpers.Utilities.log);
+    }
+
+    function getValues() {
+        Excel.run(async (context) => {
+            let sheet = context.workbook.worksheets.getActiveWorksheet();
+            let range = sheet.getRange("B2:E6");
+            range.load("values");
 
             await context.sync();
 
+            console.log(JSON.stringify(range.values, null, 4));
```

Comment:
My preference is to continue to use JSON.stringify since the output is simpler, look more structured compared to what the Firebug console will do on its own.